### ISSUE TITLE
it's odd to parse the messages between different installation ways

### ISSUE THREAD

**Issue by starndawn** [bloonix/awesant#45]
I have an old version awesant (0.9) which ships log into redis, and I want to upgrade it to 0.19 recently. The old version was installed via rpm. If I install the new version with "make && make install", the message field is named as "message" without "@" character. If it is installed via rpm, it will be named as "@message".

I also try to remove the new version with "make && make install" installation, and re-install the old version via rpm, then this time the field is named as "message" without "@" character. If I re-install the old version via "make && make install", the field is named as "@message". it's weird. 

with "@":
"{\"@timestamp\":\"2015-05-29T05:05:06-04:00\",\"@tags\":[\"nginx\"],\"@fields\":{},\"@source_path\":\"/opt/logs/access_log\",\"@source\":\"file://test12.lab.com/opt/logs/access_log\",\"@source_host\":\"test12.lab.com\",\"@message\":\"10.57.151.136 - - [25/May/2015:03:00:01 -0700] \\\"GET /ping HTTP/1.1\\\" 200 4 0.003 \\\"-\\\" \\\"-\\\" \\\"-\\\" \\\"10.5.4.2\\\" \\\"-\\\" \\\"26825245\\\" \\\"0ms\\\"\",\"@type\":\"nginx-access\"}"

without "@":
"{\"@timestamp\":\"2015-05-29T05:05:06-04:00\",\"@tags\":[\"nginx\"],\"@fields\":{},\"source_path\":\"/opt/logs/access_log\",\"source\":\"file://test12.lab.com/opt/logs/access_log\",\"source_host\":\"test12.lab.com\",\"message\":\"10.57.151.136 - - [25/May/2015:03:00:01 -0700] \\\"GET /ping HTTP/1.1\\\" 200 4 0.003 \\\"-\\\" \\\"-\\\" \\\"-\\\" \\\"10.5.4.2\\\" \\\"-\\\" \\\"26825245\\\" \\\"0ms\\\"\",\"type\":\"nginx-access\"}"

**Comment by j-schulz** [comment]
The newer awesant version uses the new logstash format, where the @ sign is not used any more.

https://github.com/bloonix/awesant/blob/master/GlobalOptions.md

See option oldlogstashjson.

**Comment by starndawn** [comment]
Yes, I know. However, after I upgrade it to the new version via rpm installation, it still uses the old logstash format. But it uses the new logstash format if the upgrade is via make&&make install.
